test/OrchardCore.Tests/Apis/Context/Extensions/HttpRequestExtensions.cs

```diff
@@ -136,12 +136,14 @@ internal static class HttpRequestExtensions
             JsonSerializerSettings settings = null)
         {
             var content = new StringContent(
-                JsonConvert.SerializeObject(value, settings ?? JsonSettings),
+                JsonConvert.SerializeObject(value, settings ?? _jsonSettings),
                 Encoding.UTF8,
                 "application/json");
 
-            var request = new HttpRequestMessage(HttpMethod.Post, requestUri);
-            request.Content = content;
+            var request = new HttpRequestMessage(HttpMethod.Post, requestUri)
+            {
+                Content = content
```

Comment:
Comma?

Reply:
Yes we can, I will ;)

Reply:
I think this one was missed

Reply:
No, it's done if you view the last updated file

https://github.com/OrchardCMS/OrchardCore/blob/3befbb21e53c6e2505a6b97131d12092b379853d/test/OrchardCore.Tests/Apis/Context/Extensions/HttpRequestExtensions.cs#L145